docs/usage_guide.rst

```diff
@@ -817,7 +817,7 @@ Adding and Removing Blockchain Accounts
 
 rotki allows to track balances of blockchain accounts.
 
-To add or modify an account navigate to the "Blockchain Balances" sub-page and click the large "+" icon. Now choose the blockchain on which you want to add an account (for now only Bitcoin, Bitcoin Cash, Ethereum, Kusama, Polkadot and Avalanche chains are supported). Then type or paste the address in the "Account" textbox and press the "Save" Button. Note that you can add multiple accounts if you click the "Add multiple addresses" checkbox and provide a comma separated list of addresses.
+To add or modify an account navigate to the "Blockchain Balances" sub-page and click the large "+" icon. Now choose the blockchain on which you want to add an account (for now only Bitcoin, Bitcoin Cash, Ethereum, Optimism, Kusama, Polkadot and Avalanche chains are supported). If the selected chain is an EVM chain (Ethereum, Optimism or Avalanche) you can also choose whether to add the address to all EVM chains. Then type or paste the address in the "Account" textbox and press the "Save" Button. Note that you can add multiple accounts if you click the "Add multiple addresses" checkbox and provide a comma separated list of addresses.
```

Comment:
Here I would split the EVM case in a new paragraph and explain that by default it's added to all chains.Also explain how we do not add it to another chain if it's a contract in mainnet etc. We have those rules written down in our docs/code but they should be in the docs too

Reply:
Modified